sdk/eventhubs/azure-messaging-eventhubs-checkpointstore-blob/src/samples/java/com/azure/messaging/eventhubs/checkpointstore/blob/EventProcessorWithOlderStorageVersion.java

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.messaging.eventhubs.checkpointstore.blob;
+
+import com.azure.core.http.HttpHeaders;
+import com.azure.core.http.policy.AddHeadersPolicy;
+import com.azure.messaging.eventhubs.EventProcessorClient;
+import com.azure.messaging.eventhubs.EventProcessorClientBuilder;
+import com.azure.storage.blob.BlobContainerAsyncClient;
+import com.azure.storage.blob.BlobContainerClientBuilder;
+import java.util.concurrent.CountDownLatch;
+
+/**
+ * This sample shows how to setup an Event Processor that uses an older version of blob storage service as
+ * checkpoint store.
+ */
```

Comment:
Consider adding more context to the situation on which a user will need to use this sample. Something like: The following sample can be used if the environment you are targeting supports a different version of Storage Blob SDK than those typically available on Azure. For example, if you are running Event Hubs on an Azure Stack Hub version 2002, the highest available version for the Storage service is version 2017-11-09. In this case, you will need to use the following code to change the Storage service API version to 2017-11-09. For more information on the Azure Storage service versions supported on Azure Stack Hub, please refer to docs.microsoft.com/en-us/azure-stack/user/azure-stack-acs-differences.